pkg/config/store/queue_test.go

```diff
@@ -139,19 +140,27 @@ func TestQueueCancelSync(t *testing.T) {
 			Value: &BackEndResource{},
 		}
 	}
+	// Wait for the queue's run loop to process chin.
+	time.Sleep(time.Millisecond)
```

Comment:
how do we know wait for 1 millisecond enough ?

Reply:
Anything else that we can do to not sleep but rather be more event driven ?

Reply:
There's no guarantee 1 msec is sufficient. There's no hook here.  That's why I added the break at line 152-156 below.
I think however, theoretically it's nicer to wait some short time before `cancel()` is called.